src/site/content/en/blog/fugu-status/index.md

```diff
@@ -0,0 +1,264 @@
+---
+title: New capabilities status
+subhead: Web apps should be able to do anything native apps can. We want to make it possible for you to build and deliver apps on the open web that have never been possible before.
```

Comment:
```suggestion
subhead: Web apps should be able to do anything native apps can. Google wants to make it possible for you to build and deliver apps on the open web that have never been possible before.
```

Reply:
I'm using "Google" here rather than "Chrome" because the first paragraph of the post says the Fugu is a "cross-company effort"